glad/generator/rust/templates/impl.rs

```diff
@@ -26,7 +34,8 @@ impl FnPtr {
             self.is_loaded = false;
         };
     }
-
+    
+    {% endif %}
     pub fn aliased(&mut self, other: &FnPtr) {
```

Comment:
It is possible that there are no aliased functions, then this is dead code, the struct or function needs `#[allow(dead_code)]`